### PR TITLE
SciPy 2020 Maintainers Track Talk

### PR DESCRIPTION
**Pull request by jacobtomlinson**
This article provides an overview of the talk I gave on the SciPy 2020 maintainers track.

It's basically just an unpacking of my [tweet thread](https://twitter.com/_JacobTomlinson/status/1280885850914553856?s=20) into a blog post. I've also included the slides via SpeakerDeck and will add the recording once SciPy folks upload it to YouTube.

### CODE REVIEW

File: _posts/2020-07-13-scipy-2020-maintainers-track.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: SciPy 2020 Maintainers Track Talk
+tagline: An update on Dask over the last 12 months
```
Comment (mrocklin) [inline _posts/2020-07-13-scipy-2020-maintainers-track.md]:
```suggestion
title: Last Year in Review
tagline: From the SciPy 2020 Maintainers Trak
```

I recommend that we flip these two.  Most readers will be more interested in the content than in the source I think.

File: _posts/2020-07-13-scipy-2020-maintainers-track.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+author: Jacob Tomlinson (NVIDIA)
+tags: [SciPy, Community, Talk]
+theme: twitter
+---
+
+{% include JB/setup %}
+
+We recently enjoyed the 2020 SciPy conference from the comfort of our own homes this year. The 19th annual Scientific Computing with Python conference was a virtual conference this year due to the global pandemic. The annual SciPy Conference brought together over 1500 participants from industry, academia, and government to showcase their latest projects, learn from skilled users and developers, and collaborate on code development.
+
+This year we had the privilege of giving the first project update on the maintainers track.
```
Comment (mrocklin) [inline _posts/2020-07-13-scipy-2020-maintainers-track.md]: There were three days I think, of which we were the last day. Regardless, I think that this bit of information isn't critical.

Comment (jacobtomlinson) [reply _posts/2020-07-13-scipy-2020-maintainers-track.md]: Ah fair enough!

**Review by jsignell — APPROVED** [review]
This looks great to me @jacobtomlinson!

**Comment by jsignell** [comment]
@jacobtomlinson, ready for publishing?

**Comment by jacobtomlinson** [comment]
Yep!

**Comment by jsignell** [comment]
I realized I don't know what that process looks like but :+1: from me for you to hit merge :)

**Comment by jacobtomlinson** [comment]
I think the process is just fixing the date in the file name and then hit merge. I'll do that now!

**Pull request merged**